layouts/New/layouts.module.css

```diff
@@ -216,3 +216,52 @@
     }
   }
 }
+
+.notFoundLayout {
```

Comment:
Can't this be reused from the home layout?

Reply:
Then the base of the home layout (shared centered content) could become a "centeredLayout"

Reply:
Just feel that if we can reuse as much stuff as possible that'd reduce the CSS bundle